src/Components/App/App.tsx

```diff
@@ -2,15 +2,25 @@ import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 import './App.scss';
 import WelcomePage from '../WelcomePage/WelcomePage';
-import Dashboard from 'Containers/Dashboard/Dashboard';
+import Dashboard from '../../Containers/Dashboard/Dashboard';
 import RoundOne from 'Containers/RoundOne/RoundOne';
+<<<<<<< HEAD
+import RoundTwo from 'Containers/RoundTwo/RoundTwo';
+=======
+>>>>>>> 78709f68b952bc755af90522a99d8121acc83743
+import Instructions from 'Containers/Instructions/Instructions';
 
```

Comment:
Merge conflict indicator looks like it decided to hang around. Please resolve and push up changes!